### ISSUE TITLE
react.types.SyntheticEvent.FormEvent missing 'target' prop

### ISSUE THREAD

**Issue by djaonourside** [kLabz/haxe-react-types#1]
@kLabz According this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1093
BaseSyntheticEvent should have target property

**Comment by djaonourside** [comment]
According this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1093
BaseSyntheticEvent should have target prop

**Comment by kLabz** [comment]
Yeah but it has been left out so that implementations can type it accordingly. In what scenario was this an issue for you?

Though maybe setting it to `TElement` would do.. I'd need to test.
